core/new-gui/src/app/workspace/component/code-editor-dialog/code-editor-dialog.component.ts

```diff
@@ -101,6 +111,17 @@ export class CodeEditorDialogComponent implements AfterViewInit, SafeStyle, OnDe
       dialog.style.width = "800px";
       dialog.style.height = "600px";
     }
+    const dialogStyle = localStorage.getItem("udfDialogStyle");
+    if (dialog !== null && dialogStyle !== null) {
+      dialog.style.cssText = dialogStyle;
+    }
+
+    const panel = document.querySelector<HTMLElement>(".cdk-overlay-pane");
+    const panelStyle = localStorage.getItem("udfPanelStyle");
+    if (panel !== null && panelStyle !== null) {
+      panel.style.cssText = panelStyle;
+    }
```

Comment:
Please document the reason we have to change CSS after the view has been initialized.